### ISSUE TITLE
Are you Okay?

### ISSUE THREAD

**Issue by godreborn** [Evilnat/Cobra-PS3#5]
this isn't really an issue, but I haven't seen you on psx-place in a while.  are you okay?  I'm worried about you.l

**Comment by Evilnat** [comment]
Yeah I'm okay, my PC crashed and I had to wait until I bought another motherboard. Thanks for worrying about me mate :)

**Comment by godreborn** [comment]
glad to hear it.  several people on both psx place and temp were worried about you too.
